src/internal/lokiutil/lokiutil.go

```diff
@@ -47,6 +48,21 @@ func forEachLine(resp loki.QueryResponse, f func(t time.Time, line string) error
 	return nil
 }
 
+func wrapEntryLog(entry streamEntry) (streamEntry, error) {
+	if strings.HasPrefix(entry.Log, "{") {
```

Comment:
I believe that this should be a full unmarshal, not just a check for starting with a curly bracket.

Reply:
Sure, yeah that's more robust